bigquery_etl/util/common.py

```diff
@@ -53,9 +53,15 @@ def random_str(length: int = 12) -> str:
     return "".join(random.choice(string.ascii_lowercase) for i in range(length))
 
 
-def render(sql_filename, format=True, template_folder="glean_usage", **kwargs) -> str:
+def render(
+    sql_filename,
+    format=True,
+    template_folder="glean_usage",
+    templates_dir="templates",
```

Comment:
Maybe make `templates_dir` optional as well, set to `templates_dir="templates"` only where applicable?

Needing to override this repeatedly with `templates_dir=""` is a bit of a code smell to me.